elasticjob-infra/elasticjob-infra-common/src/main/java/org/apache/shardingsphere/elasticjob/infra/handler/sharding/JobInstance.java

```diff
@@ -34,10 +34,16 @@
     
     private static final String DELIMITER = "@-@";
     
+    private static final String PROPERTY_JOB_INSTANCE_ID = "jobInstanceId";
+
     private final String jobInstanceId;
     
     public JobInstance() {
-        jobInstanceId = IpUtils.getIp() + DELIMITER + ManagementFactory.getRuntimeMXBean().getName().split("@")[0];
+        String jobInstIdSuffix = System.getProperty(PROPERTY_JOB_INSTANCE_ID);
```

Comment:
It's better to keep the names consistent.
```suggestion
        String jobInstanceIdSuffix = System.getProperty(PROPERTY_JOB_INSTANCE_ID);
```